js/jquery.validation.js

```diff
@@ -87,19 +87,38 @@
             $form.on('submit.validate', function(event) {
                 var $form = $(this);
 
-                // Stop the form submission.
-                event.preventDefault();
-
-                // Set up form data
-                var data = $form.serializeArray();
-               
-                // perform validation
-                $form.validation('validate', method, url, data);
+                if ( typeof FormData === "function" ) {
+                    // AJAX file upload supported
+                    // Stop the form submission.
+                    event.preventDefault();
```

Comment:
@PeterChaplin think these changes will break functionality for tlc-www and probably mla-erp too.

if `FormData` is supported I get a error with HTTP method `PUT` requests cause in validate function there is code that assumes the data variable is an array.

Also what is the behavior in regards to submit buttons ... `$form.serialiseArray()` behaviour is that the form variables for buttons are not serialized and we had to add code as a work around ... is the behaviour of `FormData` the same ... or will this be a problem ?